src/app/clusters/level-control/level-control.cpp

```diff
@@ -568,36 +572,37 @@ static EmberAfStatus moveToLevelHandler(EndpointId endpoint, CommandId commandId
     // logic is to prevent a light from transitioning from off to bright to dim.
     // Instead, a light that is off will stay off until the target level is
     // reached.
-    if (currentLevel <= state->moveToLevel)
+    if (currentLevel.Value() <= state->moveToLevel)
     {
         if (commandId == Commands::MoveToLevelWithOnOff::Id)
         {
             setOnOffValue(endpoint, (state->moveToLevel != state->minLevel));
         }
-        if (currentLevel == state->moveToLevel)
+        if (currentLevel.Value() == state->moveToLevel)
         {
             return EMBER_ZCL_STATUS_SUCCESS;
         }
         state->increasing = true;
-        actualStepSize    = static_cast<uint8_t>(state->moveToLevel - currentLevel);
+        actualStepSize    = static_cast<uint8_t>(state->moveToLevel - currentLevel.Value());
     }
     else
     {
         state->increasing = false;
-        actualStepSize    = static_cast<uint8_t>(currentLevel - state->moveToLevel);
+        actualStepSize    = static_cast<uint8_t>(currentLevel.Value() - state->moveToLevel);
     }
 
     // If the Transition time field takes the value 0xFFFF, then the time taken
```

Comment:
Please fix this comment to match the code.

Reply:
fixed